test/remote/gateways/remote_rapyd_test.rb

```diff
@@ -34,6 +34,9 @@ def setup
       'string': 'preferred',
       'Boolean': true
     }
+    @three_d_secure = {
```

Comment:
Is this used anywhere?

Reply:
No, this is an artifact where I was going to globalize the 3ds values. I might actually still do that.